lib/miq_automation_engine/service_models/miq_ae_service_vm_cloud.rb

```diff
@@ -12,5 +14,8 @@ class MiqAeServiceVmCloud < MiqAeServiceVm
     expose :floating_ips,      :association => true
     expose :security_groups,   :association => true
     expose :key_pairs,         :association => true
+    expose :associate_floating_ip
```

Comment:
these 3 should be only on OpenStack automate model, it should follow the real model placement